static/usage.html

```diff
@@ -523,7 +523,7 @@ <h2><a href="#web-browsing">Web browsing</a></h2>
                 <p>Vanadium was previously primarily focused on security hardening but we plan on
                 adding assorted privacy and usability features. In the near future, we plan to add
                 support for always incognito mode, content filtering (ad blocking, etc.), improved
-                state partitioning, backup/restore, native autofill and many other features.</p>
+                state partitioning, backup/restore, and many other features.</p>
```

Comment:
To keep consistency with other listed items, can you also remove the trailing comma?

```suggestion
                state partitioning, backup/restore and many other features.</p>
```